Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateUtil.java

```diff
@@ -245,4 +250,22 @@ public static DataHandler getDataHandlerFrom(Certificate cert) throws Certificat
         return buffer.toByteArray();
     }
 
+    public static DataHandler getDataHandlerForPem(byte[] bytes) {
+        return new DataHandler(new ByteArrayDataSource(bytes, MIME_TEXT_PLAIN));
+    }
+
+    public static PrivateKeyEntry getPrivateKeyEntry(KeyStore keystore, String alias, String password)
+        throws CertificateManagerException {
+
+        if (StringUtils.isBlank(alias) || StringUtils.isBlank(password)) {
+            throw new CertificateManagerException("Empty alias or password for the keystore.");
```

Comment:
Is it a requirement that a keystore must have a password to use CONNECT? You can have a blank password for a JKS.